src/main.rs

```diff
@@ -1,10 +1,19 @@
-pub mod db;
+pub mod auth;
 pub mod config;
+pub mod db;
 pub mod models;
+pub mod routes;
 
-use warp::Filter;
 use deadpool_postgres::Runtime;
 use tokio_postgres::NoTls;
+use warp::Filter;
+
+#[macro_export]
+macro_rules! warp_reply {
```

Comment:
This looks like it can just be a function

Reply:
having it be a macro lets you get intellisense for the consts in warp::http::StatusCode